repo2docker/buildpacks/r.py

```diff
@@ -128,12 +128,18 @@ def get_build_scripts(self):
         # This is MD5, because that is what RStudio download page provides!
         rstudio_checksum = '24cd11f0405d8372b4168fc9956e0386'
 
+        shiny_url = 'https://download3.rstudio.org/ubuntu-12.04/x86_64/shiny-server-1.5.6.875-amd64.deb'
+        shiny_checksum = 'de039a6f9e055de693524e6cb44f6a03'
+
         # Version of MRAN to pull devtools from.
         devtools_version = '2018-02-01'
 
         # IRKernel version - specified as a tag in the IRKernel repository
         irkernel_version = '0.8.11'
 
+        nbrsessionproxy_pip_spec = 'nbrsessionproxy==0.6.1'
+        nbrsessionproxy_pip_spec = 'git+https://github.com/ryanlovett/nbrsessionproxy.git@432262c'
```

Comment:
Is this duplication on purpose or a debugging left over? My preference is that we don't reference "private" forks of tools and instead use something from PyPi.

Reply:
This was only for the duration of the PR since it depended on not-yet-reviewed functionality in nbrsessionproxy. I totally agree that personal forks shouldn't be referenced in production.

I chatted with @yuvipanda a few days ago about this PR and I'm actually going to rework it. I'll probably get the new code merged to nbrsessionproxy before I complete this PR.